client/me/help/help-contact-form/index.jsx

```diff
@@ -205,15 +205,21 @@ export class HelpContactForm extends PureComponent {
 
 		wpcom.req
 			.get( '/help/qanda', { query, site } )
-			.then( ( qanda ) =>
+			.then( ( qanda ) => {
+				const sameQuestionsReturned = areSameQuestions( this.state.qanda, qanda );
+				if ( ! sameQuestionsReturned ) {
+					recordTracksEvent( 'calypso_sibyl_display_results', {
+						results_count: qanda.length,
```

Comment:
Considering the `isArray` check below, I'm wondering if this can throw and crash the whole experience. 

```suggestion
						results_count: qanda?.length,
```